### ISSUE TITLE
Bring some field formats in line with `@article`

### ISSUE THREAD

**Issue by moewew** [maieul/biblatex-bookinarticle#4]
Currently the fields `volume` and `series` (actually also `number`) inherit the default formats from `biblatex.def`, `@article`'s format differs from the standard for those fields. Since `@bookinarticle` seems closer to `@article` than say `@book` when it comes to those fields, it could be a nice idea to have

```
\DeclareFieldFormat[inarticle,bookinarticle]{series}{% series of a journal
  \ifinteger{#1}
    {\mkbibordseries{#1}~\bibstring{jourser}}
    {\ifbibstring{#1}{\bibstring{#1}}{#1}}}
\DeclareFieldFormat[inarticle,bookinarticle]{volume}{#1}% volume of a journal
\DeclareFieldFormat[inarticle,bookinarticle]{number}{#1}% number of a journal
```

to move the output of `@bookinarticle` closer to `@article`.

**Comment by maieul** [comment]
oh ! I didn't know this use of `series` field for `inarticle`. Thanks ! 

Corrections made on branch "issue5". Please check and tell me.

**Comment by moewew** [comment]
Works absolutely as expected. Thanks for the quick reaction.

**Comment by maieul** [comment]
do you have any more issue for today ? If not, I will publish it now on CTAN.

**Comment by moewew** [comment]
I can't think of anything else, no. But then I don't actually use the package in a production environment. The issue came up on TeX.SX: [Biblatex, Bibliography, and \bibnamedash for treatises published in/as articles](https://tex.stackexchange.com/questions/290972/biblatex-bibliography-and-bibnamedash-for-treatises-published-in-as-articles/291323#comment704871_290972)

**Comment by maieul** [comment]
oh, I see. @clinteastwood could open new issue if need. So I publish on CTAN

**Comment by moewew** [comment]
One minor thing I can think of though, maybe you want to add

```
\RequireBiber[3]
``` 

to make the requirement for Biber more explicit.

**Comment by maieul** [comment]
\RequireBiber[3] is for bbx / cbx file, not .sty file.

**Comment by moewew** [comment]
Mhh, I saw that as well, but I'm not sure if those who wrote the documentation thought about `biblatex` code coming in a `.sty`. It definitely works, though it doesn't quite do what I hoped, the error by `\RequireBiber[3]` comes only after `\DeclareLabelname` has already thrown an error.

**Comment by maieul** [comment]
I opened a new issue #5. I will add it to the next version of the package.
